From 81783adda59b8b813a6fe4f3fe2b3db288144a49 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Dan=20Hor=C3=A1k?= <dan@danny.cz>
Date: Fri, 3 Nov 2023 16:19:18 +0100
Subject: [PATCH] skip tests requiring mp4 support

---
 tests/test_video.py | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/tests/test_video.py b/tests/test_video.py
index 194bcab..dcd903a 100644
--- a/tests/test_video.py
+++ b/tests/test_video.py
@@ -64,7 +64,7 @@ def test_metadata(tmpdir):
     assert video.date == datetime(2020, 1, 1, 9, 0)
 
 
-@pytest.mark.parametrize("fmt", ["webm", "mp4"])
+@pytest.mark.parametrize("fmt", ["webm"])
 def test_generate_video_fit(tmpdir, fmt):
     """largest fitting dimension is width"""
 
@@ -81,7 +81,7 @@ def test_generate_video_fit(tmpdir, fmt):
     assert abs(size_dst[0] / size_dst[1] - size_src[0] / size_src[1]) < 2e-2
 
 
-@pytest.mark.parametrize("fmt", ["webm", "mp4", "ogv"])
+@pytest.mark.parametrize("fmt", ["webm", "ogv"])
 def test_generate_video_dont_enlarge(tmpdir, fmt):
     """Video dimensions should not be enlarged."""
 
@@ -96,7 +96,7 @@ def test_generate_video_dont_enlarge(tmpdir, fmt):
 
 
 @patch("sigal.video.generate_video_pass")
-@pytest.mark.parametrize("fmt", ["webm", "mp4"])
+@pytest.mark.parametrize("fmt", ["webm"])
 def test_second_pass_video(mock_generate_video_pass, fmt, tmpdir):
     """Video should be run through ffmpeg."""
     base, ext = os.path.splitext(TEST_VIDEO)
-- 
2.41.0

